src/features/dialog-manager/dialogs/ability-dialog/index.js

```diff
@@ -17,15 +17,14 @@ import incrementDexterity from './actions/increment-dexterity';
 import decrementCharisma from './actions/decrement-charisma';
 import incrementCharisma from './actions/increment-charisma';
 
-import setAbilityScores from '../../actions/set-ability-scores';
-import closeDialog from '../../actions/close-dialog';
+import confirmAbilityScoreDialog from '../../actions/confirm-ability-score-dialog';
```

Comment:
The import here doesn't match the export from the file. Seems strange to have a mismatch.

Reply:
Technically it doesn't matter from what I understand, but I agree so I've changed it anyways.